ext/standard/image.c

```diff
@@ -1148,6 +1149,100 @@ static struct gfxinfo *php_handle_webp(php_stream * stream)
 }
 /* }}} */
 
+/* {{{ php_handle_avif
+ * There's no simple way to get this information - so, for now, this is unsupported.
+ * Simply return 0 for everything.
+ */
+static struct gfxinfo *php_handle_avif(php_stream * stream) {
+	struct gfxinfo * result;
+
+	result = (struct gfxinfo *) ecalloc(1, sizeof(struct gfxinfo));
+	return result;
+}
```

Comment:
The comment appears to be out-dated.

Reply:
It's actually still true! We don't have a reliable way of getting the size of the AVIF or other information. All we do is verify (very reliably) whether an image is in fact an AVIF.

Let me know if I'm missing something here. I suspect I may be. At the very least, does the comment need to be clarified? :)

Reply:
Ah, right! Comment and implementatíon actually look good.